### PR TITLE
OSD-22522 elevate allows args with quotes

### PR DESCRIPTION
**Pull request by Tof1973**
### What type of PR is this?

_bug_

### What this PR does / Why we need it?

_ref [OSD-22522](https://issues.redhat.com//browse/OSD-22522)_

When running ocm backplane elevate with args containing single/double quotes, those args are not managed properly. Here is an exemple:
```
$ ocm-backplane elevate test -- -n openshift-operator-lifecycle-manager patch deployment packageserver \  
  --type='json' -p='[{"op": "add", "path": "/spec/template/spec/containers/0/command/-", "value": "xxx"}]' \  
  -o yaml --dry-run=client
error: unable to parse "[{op:": yaml: line 1: did not find expected node content
ERRO[0000] exit status 1 
```

### Which Jira/Github issue(s) does this PR fix?

_Resolves #[OSD-22522](https://issues.redhat.com//browse/OSD-22522)_

### Special notes for your reviewer

With this change, we can now use args with quotes as we can see here:
```
$ ./ocm-backplane elevate test -- -n openshift-operator-lifecycle-manager patch deployment packageserver \  
  --type='json' -p='[{"op": "add", "path": "/spec/template/spec/containers/0/command/-", "value": "xxx"}]' \  
  -o yaml --dry-run=client | yq .spec.template.spec.containers[0].command
- /bin/package-server
- -v=4
- --secure-port
- "5443"
- --global-namespace
- openshift-marketplace
- xxx
```

### Pre-checks (if applicable)

- [x] Ran unit tests locally
- [x] Validated the changes in a cluster

### CODE REVIEW

**Comment by Tof1973** [comment]
With this change, we can now use args with quotes as we can see here:
```
$ ~/Repositories/backplane-cli/ocm-backplane elevate test -- -n openshift-operator-lifecycle-manager patch deployment packageserver --type='json' -p='[{"op": "add", "path": "/spec/template/spec/containers/0/command/-", "value": "xxx"}]' -o yaml --dry-run=client | yq .spec.template.spec.containers[0].command
- /bin/package-server
- -v=4
- --secure-port
- "5443"
- --global-namespace
- openshift-marketplace
- xxx

```

File: pkg/elevate/elevate.go [hunk 1]
```diff
@@ -73,42 +72,24 @@ func RunElevate(argv []string) error {
...
+		err := OsRemove(tempKubeconfigPath)
+		if err != nil {
+			fmt.Println(err)
 		}
-	}
+	}()
 
 	logger.Debugln("Executing command with temporary kubeconfig as backplane-cluster-admin")
-
-	ocCmd := ExecCmd(shell, "-c", elevateCmd)
```
Comment (bmeng) [inline pkg/elevate/elevate.go:92]: Wondering why it uses `bash -c` at the first place, have you confirmed the change works on both linux and mac?

Comment (Tof1973) [reply pkg/elevate/elevate.go:92]: The reason of using bash -c was for [OSD-18058](https://issues.redhat.com/browse/OSD-18058), but as mentioned in [OSD-22522](https://issues.redhat.com/browse/OSD-22522) comment, this usage is not working as it was expecting and it is not needed as there is no need for it to have interactive prompt.

**Pull request merged**